cmake/onnxruntime_python.cmake

```diff
@@ -962,13 +962,33 @@ if (onnxruntime_USE_TENSORRT)
 endif()
 
 if (onnxruntime_USE_NV)
+  if (WIN32 OR ${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+      file(GLOB NV_LIB_FILES LIST_DIRECTORIES false "${TENSORRT_RTX_ROOT}/lib/tensorrt_*.dll"
+                                             "${TENSORRT_RTX_ROOT}/bin/tensorrt_*.dll"
+                                             "${TENSORRT_RTX_ROOT}/lib/libtensorrt_*.so.*"
+                                             "${TENSORRT_RTX_ROOT}/bin/libtensorrt_*.so.*")
+    message(STATUS "NV lib files: " ${NV_LIB_FILES})
+  endif()
```

Comment:
This will not work if TRT RTX is installed as a system package. I think this is currently not easily supported via `apt install` or something. That being said if you use [`find_path`](https://cmake.org/cmake/help/latest/command/find_path.html) instead of a file GLOB you can include system install paths and so on.